contracts/src/v0.8/functions/v1_3_0_zksync/interfaces/zksync/IGasBoundCaller.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: BUSL-1.1
```

Comment:
This feels like a vendor dependency? Is theirs BUSL? Kinda strange but possible. If this is ours, why would it be BUSL?